### ISSUE TITLE
InteractsWithMedia::getFirstMedia enforce $filters as array

### ISSUE THREAD

**Issue by vdauchy** [spatie/laravel-medialibrary#1779]
Hi,
I suppose this is a small bug as in InteractsWithMedia::class the function:

```
public function getFirstMedia(string $collectionName = 'default', array $filters = []): ?Media
    {
        $media = $this->getMedia($collectionName, $filters);

        return $media->first();
    }
```

enforce `$filters` type to `array` while `InteractsWithMedia::getMedia()` supports `$filters` as callable too.

Have a great day.

**Comment by freekmurze** [comment]
I'd accept a PR with tests that fixes this.

**Comment by freekmurze** [comment]
Fixed by #1781.
